doc/rst/basic_usage.rst

```diff
@@ -166,7 +150,7 @@ transactions. For example, consider the case where an endpoint is always expecti
 OUT, IN, OUT (such as a control transfer or a request response protocol).
 If an unplug/reset event was received after the first OUT, the host would return to sending the
 initial OUT after a re-plug, whilst the endpoint task would hang trying to send a response the IN.
-The endpoint needs to know of the bus reset in order to reset its state machine.
+The endpoint needs to know of the bus reset in order to reset it's state machine.
```

Comment:
it's should be its (it is the only possessive form that doesn't use an apostrophe as it's means it is)